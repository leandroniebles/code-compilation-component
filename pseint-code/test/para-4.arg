--lazy_syntax=0