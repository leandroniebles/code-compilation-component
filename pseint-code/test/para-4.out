=== Line 2: SynError 216
=== Line 6: SynError 72
